pikaur/srcinfo.py

```diff
@@ -130,13 +133,17 @@ def get_version(self) -> str:
         )
 
     def regenerate(self) -> None:
+        result = spawn(
+            isolate_root_cmd(
+                get_makepkg_cmd() + ['--printsrcinfo'] +
+                ['-p', os.path.basename(self.pkgbuild_path)],
+                cwd=self.repo_path
+            ), cwd=self.repo_path
+        )
+        if result.returncode != 0:
+            print_error(_("failed to generate .SRCINFO from {}:").format(self.pkgbuild_path))
+            print_stderr(result.stderr_text)
+            raise SysExit(6)
```

Comment:
ENXIO 6 No such device or address

why?

Reply:
Why any other number? I found this line https://github.com/actionless/pikaur/blob/fbb4e85847101f5ae936ba50440f37c7d0cba77b/pikaur/install_info_fetcher.py#L288
and error looked similar.
Which would be the correct one?

Reply:
i am not sure, mb

EIO 5 Input/output error

or

ENOTRECOVERABLE 131 State not recoverable